js/modules/k6/experimental/experimental.go

```diff
@@ -0,0 +1,68 @@
+// Package experimental includes experimental module features
+package experimental
+
+import (
+	"errors"
+	"time"
+
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+type (
+	// RootModule is the root experimental module
+	RootModule struct{}
+	// ModuleInstance represents an instance of the experimental module
+	ModuleInstance struct {
+		vu modules.VU
+	}
+)
+
+var (
+	_ modules.Module   = &RootModule{}
+	_ modules.Instance = &ModuleInstance{}
+)
+
+// NewModuleInstance implements modules.Module interface
+func (*RootModule) NewModuleInstance(m modules.VU) modules.Instance {
+	return &ModuleInstance{vu: m}
+}
+
+// New returns a new RootModule.
+func New() *RootModule {
+	return &RootModule{}
+}
+
+// Exports returns the exports of the experimental module
+func (mi *ModuleInstance) Exports() modules.Exports {
+	return modules.Exports{
+		Named: map[string]interface{}{
+			"setTimeout": mi.setTimeout,
+		},
+	}
+}
+
+func (mi *ModuleInstance) setTimeout(f goja.Callable, t float64) {
```

Comment:
Now that we have it as an explicit experiment could we add the expected `timeoutID` as returned value and set it always to zero?

Reply:
I would prefer if we leave this for the *next* release. Given that this won't be used by anyone.

We can have this changes in : 
1. extension
2. in that module temporarily through the development cycle

but I would argue we would probably want them globally available by next release. 

Adding unnecessary changes days before cutoff IMO just adds more ways for this to break. We currently only care about this call in order to test the event loop - anyone actually wanting to use it - shouldn't